imap_processing/swe/tests/test_swe_l1a_science.py

```diff
@@ -18,10 +18,10 @@ def decom_test_data():
     # Packet 4 has spin 3's data
     # reorder to match the order of spin in the data
```

Comment:
Do we need to be able to account for this re-ordering in code and make sure we can handle this ourselves? Or this is a one-off thing due to the way they ran their simulator?

Reply:
This is a one-off thing